plugin-server/src/worker/ingestion/event-pipeline/produceExceptionSymbolificationEventStep.ts

```diff
@@ -9,7 +9,7 @@ export function produceExceptionSymbolificationEventStep(
     const ack = runner.hub.kafkaProducer
         .produce({
             topic: runner.hub.EXCEPTIONS_SYMBOLIFICATION_KAFKA_TOPIC,
-            key: event.uuid,
+            key: String(event.team_id),
```

Comment:
We use `event.uuid` in a bunch of places. I assume this isn't some type of deduplicating key per event?

Reply:
Yup, only a partition key. Even if clickhouse used it, we use a different one when we produce from cymbal.